uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerCompanyPortal.java

```diff
@@ -71,11 +71,6 @@ public BrokerCompanyPortal() {
         localApkFileName = COMPANY_PORTAL_APK;
     }
 
-    public BrokerCompanyPortal(@NonNull final IAppInstaller appInstaller) {
-        super(COMPANY_PORTAL_APP_PACKAGE_NAME, COMPANY_PORTAL_APP_NAME, appInstaller);
-        localApkFileName = COMPANY_PORTAL_APK;
-    }
```

Comment:
Why is this code removed?

Reply:
I could not find any code referencing this one. We are installing CP using using another overload constructor. Let me know if i am missing something!

Reply:
Android Studio Find Usages feature sometimes doesn't correctly report all usages, so therefore I suggest that we add constructor back even if it is not being used. There is no harm in keeping there for now.